src/main/java/uk/gov/pay/api/app/config/StringToListConverter.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.pay.api.app.config;
+
+import com.fasterxml.jackson.databind.util.StdConverter;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class StringToListConverter extends StdConverter<String, List<String>> {
+
+    @Override
+    public List<String> convert(String value) {
+        if (StringUtils.isBlank(value)) {
+            return Collections.emptyList();
+        }
+
+        return Arrays.stream(value.split(",")).map(String::trim).collect(Collectors.toList());
```

Comment:
May be also filter by nonEmpty string to avoid list with empty strings for input like `a,,b,`

Reply:
Sure, sounds good. And it couldn't be done as separate PR, because?